server/src/domain/types.rs

```diff
@@ -104,9 +104,23 @@ macro_rules! uuid {
     };
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct Serialized(Vec<u8>);
 
+impl std::fmt::Debug for Serialized {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_tuple("Serialized")
+            .field(&String::from_utf8(self.0.clone()).unwrap_or_else(|_| {
+                format!("hash: {}", {
```

Comment:
sick idea!

Reply:
Oh, that reminds me, I wanted to fiddle with that a bit, the output is not great. I also want to have that for LdapAttributes, currently it prints the bytes for everything because of the JpegPhoto. I remember talking about it with the LDAP library author, but I don't think it got implemented.